test/quantum_mechanics/harmonic_oscillator/quantum_oscillator_wave_eigenfunctions_test.py

```diff
@@ -0,0 +1,57 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from sympy import sqrt
+from symplyphysics import (
+    assert_equal,
+    units,
+    Quantity,
+    errors,
+)
+from symplyphysics.core.symbols.quantities import evaluate_quantity
+from symplyphysics.laws.quantum_mechanics.harmonic_oscillator import wave_eigenfunctions as wave_law
+
+Args = namedtuple("Args", "n m w x")
+
+
+@fixture(name="test_args")
+def test_args_fixture() -> Args:
+    n = 1
+    m = evaluate_quantity(units.planck_mass)
```

Comment:
Why not just `Quantity`?

Reply:
In this case the scale factors of the Planck units are represented by big unevaluated fractions with `pi` and roots, and on my system, `sympy` (for some reason) takes forever to substitute them into the expression and evaluate it; it doesn't even finish and just halts. So the only thing is to evaluate them beforehand, in my opinion.